easybuild/easyblocks/n/neuron.py

```diff
@@ -216,12 +268,19 @@ def make_module_extra(self):
 
         # we need to make sure the correct compiler is set in the environment,
         # since NEURON features compilation at runtime
-        for var in ['CC', 'MPICH_CC']:
+        for var in ['CC', 'CXX', 'MPICC', 'MPICXX', 'MPICH_CC', 'MPICH_CXX']:
```

Comment:
I asked feedback in https://github.com/neuronsimulator/nrn/issues/903 about this list